February 11th Cloud Release Notes (#375)
astronomer/docs: cloud/deployment-metrics.md

The bolded value above the graph shows the maximum memory usage by a single worker at any point in time over the last 24 hours. This can help you answer, "Did any of my workers approach 100% usage of total available memory in the past 24 hours?"

## DAG Runs

You can view key metrics about recent DAG runs using the **DAGs** page in the Astronomer Cloud UI. This page shows DAG runs from the last 14 days across either all Deployments or a specific Deployment in a Workspace. For a given DAG, the **DAGs** page shows:

- Total DAG runs over the last 14 days, expressed as a bar chart. Each bar in the chart represents a DAG run: its color represents whether the DAG run was a success or a failure, while its length represents the total duration of the DAG run. If there are more than 14 DAG runs in the last 14 days, then the chart shows only the 14 most recent DAG runs

> - **Last Run End**: The ending time of the DAG's most recent DAG run, expressed relative to the current time
> - **Last Run Duration**: The duration of the DAG's most recent DAG run
> - **Average Duration**: The average Duration of all DAG runs from the last 14 days

Reviewer: ```suggestion
- **Last Run End**: The ending time of the DAG's most recent DAG run, expressed relative to the current time.
- **Last Run Duration**: The duration of the DAG's most recent DAG run.
- **Average Duration**: The average Duration of all DAG runs from the last 14 days.
```

:::info DAG Visibility

> If a DAG hasn't started a DAG run in the last 14 days, then it will not appear in the **DAGs** view.

Reviewer: ```suggestion
If a DAG run for any given DAG has not run in the last 14 days, then it will not appear in the **DAGs** view.
```
